wrappers/msbuild

```diff
@@ -66,5 +66,9 @@ arm64)
     ;;
 esac
 
+if [[ $OSTYPE == darwin* ]]; then
+    MSBUILDBINDIR=${MSBUILDBINDIR}/..
```

Comment:
This is actually a bit surprising. Since a number of releases of macOS, there is no 32 bit userland available any longer, so usually wine doesn’t support 32 bit x86 binaries there, unless jumping through a large number of hoops. I guess the recent wine improves to run all of wine as 32 and 64 bit PE binaries under one 64 bit unix integration has been enabled in the homebrew builds.

Reply:
Yes, your guess is right. Quoting from [Wine 9.0 release notes](https://gitlab.winehq.org/wine/wine/-/releases/wine-9.0#wow64)
> All modules that call a Unix library contain WoW64 thunks to enable calling
the 64-bit Unix library from 32-bit PE code. This means that it is possible to
run 32-bit Windows applications on a purely 64-bit Unix installation. This is
called the new WoW64 mode, as opposed to the old WoW64 mode where 32-bit
applications run inside a 32-bit Unix process.

Reply:
And from https://github.com/Gcenx/macOS_Wine_builds/issues/76
> These packages have supported running 32Bit windows binaries on a 64Bit only version of macOS since it became supported.

Reply:
Can you add a comment here about what this does (runs i386 binaries from the base Bin directory instead of Bin/Amd64 or Bin/Arm64), and why (briefly reference what errors you run into without this).

Reply:
Install Wine Mono 9.0 resolved 64bit MSBuild issue. This workaround is no longer needed.